.github/workflows/publish-npm-rc.yaml

```diff
@@ -0,0 +1,62 @@
+name: Publish release candidate
+
+on:
+  push:
+    branches: ['releases/[0-9]+.[0-9]+.[0-9]+-rc[0-9]+']
+
+jobs:
+  build:
+    if: github.repository == 'redwoodjs/redwood'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Required because lerna uses tags to determine the version.
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --check-files
+
+      - name: Extract version from branch
+        shell: bash
+        run: echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
+        id: extract_version
+
+      - name: Create package version
+        run: ./tasks/update-package-versions ${{ steps.extract_version.outputs.version }}
+
+      - name: Configure git credentials
+        run: |
+          git config user.name redwood-releases
+          git config user.email releases@redwoodjs.com
+      - name: Commit and tag
+        run: |
+          git commit -am "${{ steps.extract_version.outputs.version }}"
+          git tag ${{ steps.extract_version.outputs.version }}
+          git push && git push --tags
```

Comment:
🤯 that you can do this, like this.

Not sure about pushing tags though, because in reality what we're probably going to have is a workflow like this:

1. Create release branch `release/v0.32`
2. Add commits to this branch, to fix any issues / improvements for the release
3. On each commit, it should publish `0.32.0-rc.0+${SHA}`

Not sure if theres any use for each tag being present for this on the main repo

Reply:
oh ok. I misunderstood. I thought that it would follow the same process of publishing a release on the CONTRIBUTING.md, will make the changes. it's clear now, thanks.

Reply:
I think it should probably be more similar to the task in `.github/workflows/publish-npm-canary.yaml`